js/data.js

```diff
@@ -4,7 +4,8 @@ var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
-  nextEntryId: 1
+  nextEntryId: 1,
+  profile: {}
```

Comment:
I would start this out with a property for each piece of data that will be stored in it, with an initial value of `null`. That way you can look at your data model and get an immediate idea of what the shape of the data will be.